### DISCUSSION TITLE
How to add new wallet which is not existed in EIP-6963 list?

### DISCUSSION THREAD

**Discussion by 0xfrost-wonder** [WalletConnect/walletconnect-monorepo#4248] [Developer Support]
Through using the EIP-6963, I felt that was a good idea for Dapp wallet integrating developers.
But I have one problem. There are only specific wallet in the list.
If I wanna add the other one to this list, what can i do?

**Comment by glitch-txs** [comment]
hi! which libraries are you using?

**Comment by 0xfrost-wonder** [comment]
I am using @wagmi/connectors to make EIP6963 provider.

**Reply by glitch-txs** [reply]
EIP6963 will only add the wallets that you have installed

**Comment by 0xfrost-wonder** [comment]
Here is the list of available wallets.
I wanna add the wallet to this list.

**Reply by glitch-txs** [reply]
We only support this in Web3Modal, you can use wallet IDs: https://docs.walletconnect.com/web3modal/javascript/options#featuredwalletids
